src/pages/auth/signup/index.page.tsx

```diff
@@ -91,14 +91,14 @@ const SignUpPage = ({
         phone_number: data.phone_number,
         job_role: data.job_role,
         callback_url: window.location.origin + "/auth/verify/email/",
-        primary_role: "project-developer",
+        role: "project-developer",
         country: selectedTitleOption == "Select Country" ? (selectedOption as any) : null,
         program: selectedTitleOption == "Select Framework" ? (selectedOption?.toLowerCase() as any) : null
       }
     });
   };
 
-  return <SignUpForm form={form} handleSave={handleSave} loading={isLoading} primary_role={primary_role as string} />;
+  return <SignUpForm form={form} handleSave={handleSave} loading={isLoading} role={role as string} />;
```

Comment:
Is it possible to remove the `as string` from the `role` property? Since it's already defined as a string, type assertion may be unnecessary.

Reply:
I'll give it a try; I figured if it was needed before it might still be, but I'll take a look.